editor/editor_properties.cpp

```diff
@@ -115,6 +115,12 @@ void EditorPropertyText::set_string_name(bool p_enabled) {
 		Label *prefix = memnew(Label("&"));
 		prefix->set_tooltip_text("StringName");
 		prefix->set_mouse_filter(MOUSE_FILTER_STOP);
+
+		// Changes "&" label color to red. Something like commented line *should* retrieve correct color
+		Color color = Color(1, 0, 0, 1)
```

Comment:
You need to access this from the `_notification` I think, the theme might not be ready here, see the other cases